MonoGame.Framework/Graphics/GraphicsDevice.cs

```diff
@@ -159,6 +159,20 @@ public GraphicsAdapter Adapter
             private set;
         }
 
+        internal Metrics _metrics;
+
+        /// <summary>
+        /// The rendering data for debugging and profiling. 
+        /// </summary>
+        public Metrics Metrics { get { return _metrics; } }
+
+        internal bool _enableMetrics = true;
+
+        /// <summary>
+        /// Enables receiving metrics data. Disabling it is a small performance hint. <c>true</c> by default.
+        /// </summary>
+        public bool EnableMetrics { get { return _enableMetrics; } set { _enableMetrics = value; } }

```

Comment:
I wonder if it is worth disabling them at all.  We're talking about very little overhead right now.